pkg/state/finalization.go

```diff
@@ -0,0 +1,85 @@
+package state
+
+import (
+	"fmt"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/pkg/errors"
+	"github.com/wavesplatform/gowaves/pkg/proto"
+)
+
+const finalizationKey = "finalization"
```

Comment:
All keys have to be introduced in https://github.com/wavesplatform/gowaves/blob/8b0ac7798b242cf9db7cea33f7691bb1d80e7331/pkg/state/keys.go#L139

Reply:
Fixed